### ISSUE TITLE
Azure Pipelines is not detected

### ISSUE THREAD

**Issue by vtbassmatt** [watson/ci-info#22]
**Details:**
 - Ci name: Azure Pipelines
 - Website: https://azure.microsoft.com/services/devops/pipelines/
 - Default env variables: https://docs.microsoft.com/azure/devops/pipelines/build/variables

**Environment variables**

<details>
<summary>Environment variables example</summary> 

```
// Add a list of actual environment variables available to the build here
// Please keep the blank line above and below the code block
// WARNING: Be careful not to include any private or otherwise sensitive information
```

</details>

**Additional context**
Disclosure: I'm a product manager on Azure Pipelines.

**Comment by DSchau** [comment]
@vtbassmatt would you be able to PR your changes in f01fe13 so we can get azure pipelines support? We're using azure pipelines over at gatsbyjs/gatsby and we'd love the ability to detect CI for Azure Pipelines just like we do for Travis/CircleCI/etc. :)

**Comment by watson** [comment]
Thanks for your PR #23 - Azure Pipelines will now be supported as of the next release which I will make shortly 💯

**Comment by watson** [comment]
Support for Azure Pipelines has just been released in v2.0.0
